code/modules/research/xenoarchaeology/artifact/artifact_vendor.dm

```diff
@@ -290,6 +290,7 @@ var/static/list/badstuff2putin = list(
 		/mob/living/simple_animal/cricket = 1,
 	)
 	var/objprob = 10
+	flags = FPRINT  | OPENCONTAINER | NOREACT | SILENTCONTAINER
```

Comment:
why is this an open container?

Reply:
because I stole the flags from cans. You're right. Updated. I changed the other soda_can/artifact to OPENCONTAINER

Reply:
why is this noreact? maybe ideally i'd like to see the can react when it opens, that sounds fairer.

Reply:
I just stole the flags from the cans because it'd throw an error. What would you like to see for flags?

Reply:
SILENTCONTAINER is the only one necessary, but I assume you just forgot to put the flags

Reply:
None of them are really necessary with the change to the unit test, unless there's another problem I don't know about

Reply:
you're right and hopefully some redundancy can't hurt